charms/kfp-ui/metadata.yaml

```diff
@@ -3,12 +3,13 @@ summary: Reusable end-to-end ML workflows built using the Kubeflow Pipelines SDK
 description: |
   Machine learning (ML) toolkit that is dedicated to making deployments
   of ML workflows on Kubernetes simple, portable, and scalable.
-min-juju-version: "2.9.0"
-series: [kubernetes]
+containers:
+  ml-pipeline-ui:
+    resource: ml-pipeline-ui
 resources:
-  oci-image:
+  ml-pipeline-ui:
```

Comment:
I'd rather we do not change oci-image to keep it uniform for our scripts.
If we change it, image retrieval script will need to be updated as well.

Reply:
I'm not too opinionated on this, but I don't think we can rely on this always being named `oci-image` because of our [multi-container charms](https://github.com/canonical/kubeflow-profiles-operator/blob/e46577ac6d7c7d97c5ddc3e6443a1749f006d305/metadata.yaml#L16).  I wonder if we can make the image retrieval script collect all resources?

I think some of our single-container charms also have [specific names here](https://github.com/canonical/kubeflow-tensorboards-operator/blob/bf34589af8fd8e144c9df299ca5f7724a560d3d5/charms/tensorboard-controller/metadata.yaml#L14)